turbopack/crates/turbopack-wasm/src/module_asset.rs

```diff
@@ -191,15 +191,15 @@ impl ChunkItem for ModuleChunkItem {
         self.module.ident()
     }
 
-    #[turbo_tasks::function]
-    fn references(&self) -> Vc<ModuleReferences> {
-        let loader = self
-            .module
-            .loader()
-            .as_chunk_item(Vc::upcast(*self.chunking_context));
-
-        loader.references()
-    }
+    // #[turbo_tasks::function]
+    // fn references(&self) -> Vc<OutputAssets> {
+    // let loader = self
+    //     .module
+    //     .loader()
+    //     .as_chunk_item(Vc::upcast(*self.chunking_context));
+
+    // loader.references()
+    // }
 
```

Comment:
```suggestion
```

You want to change the `WebAssemblyModuleAsset::references` to return a single reference to the `loader()`.

Something like that:

``` diff
     #[turbo_tasks::function]
     fn references(self: Vc<Self>) -> Vc<ModuleReferences> {
-        self.loader().references()
+        SingleChunkableModuleReference::new(self.loader())
     }